### PR TITLE
[T] (quests) a_dark_forest 

### PR DESCRIPTION
**Pull request by Ricky-fight**
Co-authored-by: Autumn秋殇 <2924297411@qq.com>
Co-authored-by: Rasilia-冬风 <2634779867@qq.com>
Co-authored-by: lailai <m18721693124@163.com>
#8

### CODE REVIEW

**Review by xinyihl — APPROVED** [review]

File: config/ftbquests/quests/chapters/a_dark_forest.snbt [hunk 1]
```diff
@@ -2412,7 +2412,7 @@
 				{
 					id: "37A858371841CDE7"
 					type: "item"
-					title: "Decorative armors"
+					title: "装饰用装备"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/a_dark_forest.snbt:2415]:
```suggestion
					title: "装饰性装备"
```

File: config/ftbquests/quests/chapters/a_dark_forest.snbt [hunk 2]
```diff
@@ -2300,17 +2300,17 @@
...
 			y: 6.5d
 			shape: "square"
-			subtitle: "Make some armor"
+			subtitle: "制作一些装备"
 			description: [
-				"I will not die here. That is not an option."
+				"我不会死在这。这压根不是一个选择。"
 				""
-				"Oddly, &cbasic armors do not seem to last very long&r. They protect me well enough, but they shatter much too quickly."
+				"奇怪的是，&c原版装备在这里都很脆弱&r。他们能很好的保护我，但是过不了多久就碎了。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/a_dark_forest.snbt:2307]:
```suggestion
				"奇怪的是，&c原版装备在这里都很脆弱&r。虽然目前它们足够能保护我，但是过不了多久就碎了。"
```

File: config/ftbquests/quests/chapters/a_dark_forest.snbt [hunk 3]
```diff
@@ -707,18 +707,18 @@
 			]
 		}
 		{
-			title: "What're Ya Buyin'?"
+			title: "想来点什么？"
 			x: 6.5d
 			y: 4.5d
-			subtitle: "These pig men have a healthy respect for wealth"
+			subtitle: "这些猪人对财富有一种健康的尊重"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/a_dark_forest.snbt:713]:
```suggestion
			subtitle: "这些猪人对财富有一种合理的尊重"
```
https://www.hujiang.com/c/k_4517587/

File: config/ftbquests/quests/chapters/a_dark_forest.snbt [hunk 4]
```diff
@@ -707,18 +707,18 @@
...
 			description: [
-				"I've found a primitive tribal society, whom I've named the &bPiglins&r."
+				"我发现了一个原始社会，我把他们命名为&b猪灵&r。"
 				""
-				"While normally hostile, I've found them to be &6completely fascinated with gold&r."
+				"尽管他们通常充满敌意，但我发现他们&6对黄金异常着迷&r。"
 				""
-				"They do not attack me while I wear Gold, and I can even &btrade with them&r by tossing some Ingots their way."
+				"当我穿着金制装备时他们并不会攻击我。我甚至还可以向他们扔金锭来与他们&b进行贸易&r。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/a_dark_forest.snbt:719]:
```suggestion
				"当我穿着金制装备时他们并不会攻击我。我甚至还可以向他们扔金锭来与他们&b进行交易&r。"
```

File: config/ftbquests/quests/chapters/a_dark_forest.snbt [hunk 5]
```diff
@@ -707,18 +707,18 @@
...
 				""
-				"I've noted a list of trades available with these piglins. I can check the items they will trade me by viewing the recipe for a &6gold ingot&r in JEI and looking at the &6information tab&r."
+				"我记下了一份贸易清单。通过在JEI中查看&6金锭&r的配方和&6信息选项卡&r，我可以查看他们想要交易的物品。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/a_dark_forest.snbt:721]:
```suggestion
				"我记下了一份交易清单。通过在JEI中查看&6金锭&r的配方和&6信息选项卡&r，我可以查看他们想要交易的物品。"
```
翻译用词尽量统一。

File: config/ftbquests/quests/chapters/a_dark_forest.snbt [hunk 6]
```diff
@@ -883,23 +883,23 @@
...
+			subtitle: "要发财了！"
 			description: [
-				"There's not much of immediate use in this horrid wasteland, but there's a shocking amount of &bGold&r!"
+				"这个可怕的荒地目前并没有什么作用，但是这边黄金的量特别的多。"
 				""
-				"My immediate thought was something along the lines of &o\"gimme gimme gimme\"&r before remembering where exactly I am."
+				"在我想起来我在哪之前，我的第一反应是“gimme, gimme, gimme”之类的东西。"
 				""
-				"Might as well grab some anyway, for... sentimental reasons."
+				"不管怎样，顺便拿走一些吧，因为……多愁善感。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/a_dark_forest.snbt:895]:
```suggestion
				"不管怎样，顺便拿走一些吧，因为……我乐意。"
```

File: config/ftbquests/quests/chapters/a_dark_forest.snbt [hunk 7]
```diff
@@ -1098,23 +1098,23 @@
...
 			y: 11.5d
 			shape: "hexagon"
-			subtitle: "Hedge Magic, but it's a start."
+			subtitle: "虽然很简单且基础，但是货真价实的魔法。"
 			description: [
-				"With this Rose Gold in hand, I can start making some &6tinctures and concoctions&r in a &bMixing Cauldron&r."
+				"有了这些玫瑰金，我就可以使用&b混合锅&r来制作一些&6混合物和调剂&r。"
 				""
-				"I can distill what little life energies exist in the flora of this place into something more useful to me."
+				"我可以将这地方的植被的一点点生命能力变成对我而言更重要的东西。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/a_dark_forest.snbt:1115]:
```suggestion
				"我可以将这地方的植被的一点点生命能变成对我而言更重要的东西。"
```

**Pull request merged**